app/controllers/projects_controller.rb

```diff
@@ -101,16 +101,21 @@ def toggle_status
   def sign_user_out
     user = User.find(params[:user_id])
     @project = Project.find(params[:id])
-    if can?(:edit, @project) || current_user == user
-      @project.remove_user(user)
-      if user == current_user
-        redirect_to projects_path
+    if not (user.is_wimi? and @project.wimis.count <= 1)
```

Comment:
Hm, ok. Ich weiß nur, dass ich oftmals darauf hingewiesen wurde, unless statt if not zu verwenden. Von unless else weiß ich nichts...